x/cosmwasmpool/model/store_model.go

```diff
@@ -96,3 +96,8 @@ func (p CosmWasmPool) GetStoreModel() poolmanagertypes.PoolI {
 func (p CosmWasmPool) SetWasmKeeper(wasmKeeper types.WasmKeeper) {
 	panic("CosmWasmPool.SetWasmKeeeper not implemented")
 }
+
+// GetPoolDenoms implements types.PoolI.
+func (p *CosmWasmPool) GetPoolDenoms(ctx sdk.Context) []string {
+	panic("CosmWasmPool.GetPoolDenoms not implemented")
```

Comment:
Can we have it Return empty string for now?
Concerned that we might call this API from higher level (pool manager side or even worse somehow called in begin or end blocker), leaving us to panic unexpectedly

Reply:
This is okay, we panic on every store model here as we do not intend to use any of them directly here